skops/card/_model_card.py

```diff
@@ -380,6 +380,13 @@ def _generate_card(self) -> ModelCard:
         # add evaluation results
 
         template_sections = copy.deepcopy(self._template_sections)
+        if self.metadata:
+            model_file = self.metadata.to_dict().get("model_file")
+            if model_file:
+                template_sections["get_started_code"] = (
+                    f"import pickle\nwith open({model_file}, 'rb') as file:\n    clf ="
+                    " pickle.load(file)"
```

Comment:
I think we can also add a `predict` example here, with the data available in the config file.

Reply:
Addressed. new version of card: https://huggingface.co/merve/test-code-generation